bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -0,0 +1,144 @@
+# OpenTherm Gateway binding
+
+This binding is used to integrate the OpenTherm Gateway into openHAB2.
+The OpenTherm Gateway is a module designed by Schelte Bron that is connected in between a boiler and a thermostat and communicates using the OpenTherm protocol.
+
+More information on the OpenTherm Gateway device can be found at http://otgw.tclcode.com/
+
+Comments on this binding can be submitted to the openHAB community topic at https://community.openhab.org/t/39160
+
+## Supported Things
+
+The OpenTherm Gateway binding currently only supports one thing, and that's the gateway itself.
+
+## Discovery
+
+The binding does not support auto discovery.
+
+## Binding Configuration
+
+The binding itself does not require any configuration.
+
+## Thing Configuration
+
+The binding is designed to support various ways of connecting to the OpenTherm Gateway device, but currently only supports a TCP socket connection.
```

Comment:
Mine was too far from my openHAB instance I have it directly connected via MQTT now :-)

Never mind, I didn't realize it was serial over TCP :-)